### PR TITLE
Reviews update

### PR DESCRIPTION
**Pull request by chrimack**
@AlexArms @kate-darby @sunkim0330 Most of these commits are styling, but a few were to add some functionality or fix bugs.

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -79,9 +83,9 @@ const App = () => {
   return !product.id || !styles.length || !reviewMetaData.product_id ? <div>Loading Epic Shopping Xperience...</div> : (
     <div>
       Super Fun Shopping Experience
-      <Overview product={product} styles={styles} defaultStyle={defaultStyle} totalReviews={totalReviewCount} averageRating={averageRating}/>
+      {/* <Overview product={product} styles={styles} defaultStyle={defaultStyle} totalReviews={totalReviewCount} averageRating={averageRating}/>
```
Comment (AlexArms) [inline client/src/App.jsx]: These are going to need to be uncommented 😆

**Review by sunkim0330 — APPROVED** [review]
nice work on stylings 😳👏👏

**Review by AlexArms — CHANGES_REQUESTED** [review]
Look good to me aside from the comment I made about needing to uncomment the other components in App.jsx. When I did a test merge with a test branch it did create a conflict. Your component seeming offset to the left. I was able to fix that by just throwing an auto margin on it, and commenting out the other one (in your Grid styled component, not sure if this broke anything else. Looked good to me).

**Review by AlexArms — APPROVED** [review]
👍

**Pull request merged**